types/twitter-text/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for twitter-text v1.14.7
+// Type definitions for twitter-text v2.0.0
 // Project: https://github.com/twitter/twitter-text
-// Definitions by: rhysd <https://rhysd.github.io>
+// Definitions by: rhysd <https://github.com/rhysd>
```

Comment:
I think you should not change this line. I did not receive ping comment from bot.

And please add your signature for maintaining v2.

Reply:
I changed this because of this review comment: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/22262#discussion_r158871577 (I think the old line was why you didn't get pinged)

Also I already maintain several dozen github repos, I don't have the capacity to commit to maintaining anything else 😄

Reply:
> I think the old line was why you didn't get pinged

I don't think so. Please see [this PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/21091). Bot sends ping to me. But bot's behavior may be changed. So let's see what happen on next PR. Anyway, I don't want to block this PR by this :+1: